Source/WebCore/Configurations/WebCore.xcconfig

```diff
@@ -196,6 +200,7 @@ SECTORDER_FLAGS_Production[sdk=macosx*] = -Wl,-order_file,WebCore.order;
 
 EXCLUDED_SOURCE_FILE_NAMES = pdfjs*/*;
 EXCLUDED_SOURCE_FILE_NAMES[sdk=iphone*] = $(inherited) *.tiff *Cursor.png npapi.h npfunctions.h npruntime.h npruntime_internal.h;
+EXCLUDED_SOURCE_FILE_NAMES[sdk=xr*] = $(inherited) *.tiff *Cursor.png npapi.h npfunctions.h npruntime.h npruntime_internal.h;
```

Comment:
We still have the `np` files?!

Reply:
Doesn't look like we do. These names were just left behind from the headers were removed in 2021 in <https://commits.webkit.org/243690@main>.